### DISCUSSION TITLE
System requirements needs to be updated

### DISCUSSION THREAD

**Discussion by chrisbadley** [discord-tickets/bot#460] [General]
System requirements for self hosted bot are incorrect and need updating.

"At least 150MB RAM" as per the current requirements, however I installed Discord Tickets 4.0.6 and it was immediately showing as consuming 224.4mb of RAM on pm2 list.

In any case, great free Discord ticketing system, thanks for your time in sharing it.

Chris

**Comment by eartharoid** [comment]
@chrisbadley memory usage depends on the number of guilds your bot is in and mostly the number of members.
It can use less than 100MB but 120-150MB is more realistic for an average small server.

How many members do you have in total?

**Reply by chrisbadley** [reply]
1 guild with 1000 members.
